test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -217,6 +219,119 @@ public void ConfigGetConfigKeyCommand_HelpMessage_Success()
             DotnetCliUtil.VerifyResultSuccess(result, helpMessage);
         }
 
+        [Theory]
+        [InlineData("signatureValidationMode", "accept")]
+        [InlineData("maxHttpRequestsPerSource", "64")]
+        public void ConfigSetCommand_AddNewConfigSettingWithConfigFileArg_Success(string key, string value)
+        {
+            // Arrange & Act
+            using var testInfo = new TestInfo("NuGet.Config");
+            var filePath = Path.Combine(testInfo.WorkingPath, "NuGet.Config");
+
+            var result = CommandRunner.Run(
+                DotnetCli,
+                Directory.GetCurrentDirectory(),
+                $"{XplatDll} config set {key} {value} --configfile {filePath}",
+                waitForExit: true);
+
+            var settings = Configuration.Settings.LoadDefaultSettings(
+                testInfo.WorkingPath,
+                configFileName: filePath,
+                machineWideSettings: new XPlatMachineWideSetting());
+
+            var configSection = settings.GetSection("config");
+            var values = configSection?.Items.Select(c => c as AddItem).Where(c => c != null).ToList();
```

Comment:
Tests should be deterministic and always produce the same result so configSection should always have a result or should always be null, which means we should not allow configSection to be null in some cases and handle it quietly in the test.

If configSection can be null in some cases, and we don't have a test that covers that, or we want to add a test for how this config set command will handle that scenario, we should add a separate test for that.

Reply:
`configSection` should not be null in any of these tests, since I'm only accessing it after adding a value to it. Is it sufficient to remove the `?` suffix, or do I need add an assertion that `configSection` is not null as well?